### PR TITLE
fix(jdk issue): adding case where artifacts starts from AMI

### PR DESCRIPTION
**Pull request by fgelcer**
following commit 4aa15e3b43c4b2bfe4cdc0df01cbc0c77c1d804c we still
needed
to address case where we start from an image (AMI), so
in here, we are adding it to the beginning of the upgrade
test, so we can go over each node and install SDKMAN.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fgelcer** [comment]
we saw it passing [in here](https://jenkins.scylladb.com/view/enterprise-2019.1/job/enterprise-2019.1/job/rolling-upgrade/job/rolling-upgrade-centos7/32/)

File: sdcm/fill_db_data.py [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ class FillDatabaseData(ClusterTester):
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 2,
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 3,
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 4,
-                        "#REMOTER_RUN sudo nodetool flush",
```
Comment (fruch) [inline sdcm/fill_db_data.py:1011]: why those are in the same commit ?

Comment (fgelcer) [reply sdcm/fill_db_data.py:1011]: they were missing from previous related commit, and once the test really started running (apart from all the hassle to have 2019.1 jobs running), i figured out that it was missing.

BTW, once 2019.1.22 will be released, we will revert this (and the previous one)

**Review by ShlomiBalalis — APPROVED** [review]

File: sdcm/fill_db_data.py (hunk 1 above)
Comment (fruch) [reply sdcm/fill_db_data.py:1011]: not sure we have a separate branch for the upgrades anyow

**Pull request merged**